### PR TITLE
fix(CLNP-2719): when frozen or muted, input should be disabled

### PR DESCRIPTION
**Pull request by bang9**
## External Contributions

This project is not yet set up to accept pull requests from external contributors.

If you have a pull request that you believe should be accepted, please contact
the Developer Relations team <developer-advocates@sendbird.com> with details
and we'll evaluate if we can setup a [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) to allow for the contribution.

## For Internal Contributors

[CLNP-2719]

## Description Of Changes

Add a brief description of the changes that you have involved in this PR

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [x] Bugfix
- [ ] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [ ] Improvement (refactor code)
- [ ] Test

[CLNP-2719]: https://sendbird.atlassian.net/browse/CLNP-2719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/uikit-react-native/src/domain/groupChannel/component/GroupChannelInput.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const GroupChannelInput = ({ inputDisabled, ...props }: GroupChannelProps['Input
       keyboardAvoidOffset={keyboardAvoidOffset}
       inputMuted={chatAvailableState.muted}
       inputFrozen={chatAvailableState.frozen}
-      inputDisabled={inputDisabled ?? chatAvailableState.disabled}
+      inputDisabled={chatAvailableState.disabled ? true : inputDisabled}
```
Comment (bang9) [inline packages/uikit-react-native/src/domain/groupChannel/component/GroupChannelInput.tsx:31]: `chatAvailableState.disabled = muted || frozen` 입니다.

Comment (HoonBaek) [reply packages/uikit-react-native/src/domain/groupChannel/component/GroupChannelInput.tsx:31]:
```suggestion
      inputDisabled={chatAvailableState.disabled || inputDisabled}
```
How about this?

**Review by HoonBaek — APPROVED** [review]
LGTM!

File: packages/uikit-react-native/src/domain/groupChannel/component/GroupChannelInput.tsx (hunk 1 above)
Comment (bang9) [reply packages/uikit-react-native/src/domain/groupChannel/component/GroupChannelInput.tsx:31]: The result is the same, but I've expressed the logic in a way that might be a bit more intuitive when read the code.

**Pull request merged**